src/color-system.mdx

```diff
@@ -0,0 +1,94 @@
+
+## Usage
+### Using CSS variables
+We have not used HEX values to create the CSS variables for each color. They are instead written in RGB digits, without parentheses. For example `--color-blue-default: 33, 150, 243;` (which would be `#2196f3` in HEX). The reason for such a choice is empowering you to write the colors with alpha channel if needed.
+
+Thus, to get a solid blue color, you would write `color: rgb(var(--color-blue-default));`.  
+To get a blue color with 10% opacity, write `color: rgba(var(--color-blue-default), 0.1);` instead.
+
+### Available palettes
+In your web-components, you only need to import one of the palettes. Importing a palette makes all included CSS variables available in that web-component. Which one you should import depends on how many colors you need for that specific component. If you need more colors than what is offered in the UI palette, then simply remove that and import a palette that suits your component better.
+
+Each palette also includes Lime Technologies' brand colors. So no matter which one you import, you will always have access to our brand colors and can use them by their specific CSS variable names.
+
+#### **UI palette** (default)
+Includes 10 color swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies brand colors.
+
+<limel-example-ui-color-palette />
+
+In most cases, importing the UI palette is enough and gives you colors which you normally need in a user interface. This palette includes a few hand picked swatches that add an extra layer of meaning to different elements. Using these, an element in the UI can quickly communicate to the user what it does or indicates.
+
+In this palette, there is a `-default` and `-light` swatch available for each hue. The `-light` tints are usually good to use when creating interactive visual feedback. For example a button that has `--color-green-default` as background color, can get `--color-green-light` when it is hovered by the cursor.
+___
+
+#### **Primary palette**
+Includes 40 color swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies brand colors.
+<limel-example-primary-color-palette />
+
+Sometimes, just a handful of colors is not enough and you need a bit more. For example when you need colors for your icons, or diagrams. This palette is designed to provide a decent number of harmonic colors while easing the choice of colors for you. Used for primary purposes and meaningful communications with colors.
+
+Note that this palette also includes brand colors, and contrast swatches.
+___
+
+#### **Extended palette**
+Includes 100 color swatches (illustrated below) + 19 contrast swatches + 11 Lime Technologies brand colors.
+
+<limel-example-extended-color-palette />
+
+This palette offers a much richer set of colors that can be used for visual communication, iconography, illustrations, and charts.
+___
+
+## Lime Technologies Brand Colors
```

Comment:
Lime Technologies Sweden AB, oh lord! You're right :D very beautiful and long name. :P and there are Norway, Denmark, Finland, Jävle and probably more :)

But since their common section of the name is "Lime Technologies", I suggest we may better keep using that. Lime alone may remind other companies. Like those Lime bikes and many other companies with Lime in their names. :/

When we internally say "Lime" it's so obvious what we're referring to. But externally, this will be certainly a confusion.

Reply:
Last thing I heard, we were instructed to say only "Lime" externally too. But that might have changed since the Lime scooters appeared I guess… could you ask someone at marketing about what we should use?

Reply:
sure. I'll ask

Reply:
Never got an answer on this. So let's be in the safe side and write the full name for now.